qiskit/aqua/components/neural_networks/quantum_generator.py

```diff
@@ -308,6 +316,46 @@ def objective_function(params):
 
         return objective_function
 
+    def _convert_to_gradient_function(self, gradient_object, quantum_instance, discriminator):
+        """
+        Convert to gradient function
+
+        Args:
+            gradient_object (Gradient): the gradient object to be used to
+                compute analytical gradients.
+            quantum_instance (QuantumInstance): used to run the quantum circuit.
+            discriminator (torch.nn.Module): discriminator network to compute the sample labels.
+
+        Returns:
```

Comment:
Hi @woodsp-ibm ,

These couple lines of code seem to be the source of the linting error that the checks threw - I simply followed the format used with the function [*_get_objective_function*](https://github.com/Qiskit/qiskit-aqua/blob/5cc28c779e2251dd602ecfeeb59c292848386338/qiskit/aqua/components/neural_networks/quantum_generator.py#L282-L292) - hopefully that's okay?

Let me know if you'd like me to work on anything else before you can approve the PR!